app/Http/Controllers/JobSummaryController.php

```diff
@@ -29,14 +29,40 @@ public function show(Request $request, JobPoster $job)
 
         $summaryLang = Lang::get('hr_advisor/job_summary');
 
+        $job_review_data = [
+            'imgSrc' => '/images/job-process-summary-tool-edit.svg',
+            'imgAlt' => "{$summaryLang['edit_poster_icon']} {$summaryLang['flat_icons']}",
+            'text' => $summaryLang['edit_poster_button'],
+            'url' => route('hr_advisor.jobs.review', $job),
+            'disabled' => false,
+        ];
+
+        $job_preview_data = [
+            'imgSrc' => '/images/job-process-summary-tool-view.svg',
+            'imgAlt' => "{$summaryLang['view_poster_icon']} {$summaryLang['flat_icons']}",
+            'text' => $summaryLang['view_poster_button'],
+            'url' => '/',
```

Comment:
@tristan-orourke Okay, all you need to do is add the route here when its ready

Reply:
Awesome, thanks.